src/model.cpp

```diff
@@ -46,6 +46,7 @@ void Model::ImportFromFile(const std::string &filename) {
   int bcc_ext = strcmp(fileext.c_str(), ".bcc");
   if (bcc_ext == 0) {
     ImportBccFile(filename);
+    segment_type_ = GetCurves()[0].get_segment_type();
```

Comment:
Please add a guard for the case where this has no curves -- right now it would segfault when you attempt to access [0] for an empty model.

e.g.
```
if (!(GetCurves().empty())) segment_type_ = GetCurves()[0].get_segment_type();
```